proxy.go

```diff
@@ -105,7 +111,16 @@ func (p *Proxy) allowedResponse(w http.ResponseWriter, r *http.Request) {
 	p.prependBucketToHostPath(r)
 
 	username := fmt.Sprintf("%v", claims["sub"])
-	filepath := strings.Replace(r.URL.Path, "/"+p.s3.bucket+"/", "", 1)
+	rawFilepath := strings.Replace(r.URL.Path, "/"+p.s3.bucket+"/", "", 1)
+
+	filepath, err := helper.FormatUploadFilePath(rawFilepath)
+	if err != nil {
+		log.Debugf(err.Error())
+		p.notAcceptableResponse(w, r)
```

Comment:
since this function is basically just one line - I'd prefer to just write the header directly instead of having an extra function.

Reply:
I see the reasoning, I only chose to write it this way because all header responses in the `proxy` are done with one line functions, too. So this keeps the code consistent imo :)

Reply:
I managed to overcome my consistency constraints, so now this is done.